Documentation/ceph-monitoring.md

```diff
@@ -210,3 +210,30 @@ labels:
     prometheus: k8s
 [...]
 ```
```

Comment:
It would be nice to have a table of contents at the top so new topics like this can be more easily discovered, but that is really a separate question from this PR.